backend/src/v4/models/sequenceActivities.js

```diff
@@ -241,7 +241,7 @@ SequenceActivities.get = async (account, model, sequenceId) => {
 		activities = await FileRef.getSequenceActivitiesFile(account, model, utils.uuidToString(sequenceId));
 	} catch(e) {
 		activities = await createActivitiesTree(account, model, sequenceId);
-		FileRef.storeFile(account, activityCol(model) + ".ref", account, utils.generateUUID({string: true}), JSON.stringify(activities),  {"_id": sequenceId});
+		await FileRef.storeFile(account, activityCol(model) + ".ref", account, utils.generateUUID({string: true}), JSON.stringify(activities),  {"_id": sequenceId});
```

Comment:
did you need await here for a test?

The whole point is the user doesn't need to wait for it to be cached, even if it failed, we don't care. that's why we're not awaiting here.

Reply:
@carmenfan This is what was causing the sequence activities tests to fail. I'll see what else can be done here.